sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,67 @@
 Version history
 ===============
 
+6.10.0 (2022 May)
+-----------------
+
+New file formats:
+
+* Slidebook 7
+   - added support for reading SlideBook Format 7 from SlideBook 2021
+     This functionality was implemented and contributed by Intelligent Imaging Innovations
+
+File format fixes and improvements:
+
+* CellWorX / MetaXpress
+   - channel metadata will now be preserved for all channels by reading from each file
+
+* Cellomics
+   - marked the private Pattern field as transient to fix memo file generation with JDK17
+
+* DeltaVision
+   - fixed detection of incorrect XY tile counts
+
+* Hitachi S-4800
+   - enhanced format recognition and identification of datasets
+   - corrected the units for physical sizes to nanometers instead of micrometers
+
+* Nikon NIS-Elements ND2
+   - improved metadata parsing, especially for objective data, timestamps, and exposure times
+
+* PerkinElmer Operetta
+   - improved checks for invalid TIFFs and supplemental metadata files
+   - updated image names to make well names more readable
+
+* PerkinElmer Vectra QPTIFF
+   - reader has been updated to support schema version 4
```

Comment:
Require the acknowledgement from https://github.com/ome/bioformats/pull/3829#issuecomment-1136072908. Also separate discussion about whether the format should be simply renamed as `Vectra QPTIFF` or more simply `QPTIFF` as per https://www.businesswire.com/news/home/20181031005857/en/Akoya-Biosciences-Acquires-Phenoptics-Portfolio-from-PerkinElmer